config/database.js

```diff
@@ -35,43 +36,64 @@ module.exports = {
     useNullAsDefault: true
   },
 
-  /*
-  |--------------------------------------------------------------------------
-  | MySQL
-  |--------------------------------------------------------------------------
-  |
-  | Here we define connection settings for MySQL database.
-  |
-  | npm i --save mysql
-  |
-  */
   mysql: {
     client: 'mysql',
     connection: {
       host: Env.get('DB_HOST', 'localhost'),
+      port: Env.get('DB_PORT', 3306),
       user: Env.get('DB_USER', 'root'),
       password: Env.get('DB_PASSWORD', ''),
-      database: Env.get('DB_DATABASE', 'adonis')
+      database: Env.get('DB_DATABASE', 'adonis'),
+      debug: false
+    }
+  },
+
+  maria: {
```

Comment:
i know, but i added as it was mentioned in the docs, so i thought maybe there is something different